### DISCUSSION TITLE
rAudio 1 - Official Release

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#21] [General]
As of 20201224, **rAudio 1** was officially released for public.
All image files have been updated to the latest.

Thank you all beta testers for you contributions.

**Comment by cYCL157** [comment]
Thanks for all of the hard work @rern !!

**Comment by ottl05** [comment]
thanks

**Comment by Roland-B** [comment]
Amazing work, thanks!

**Comment by chipdipru** [comment]
Hello! Could you please update "system-i2s.json" file by adding string
"ChipDip DAC": "chipdip-dac",
to add ChipDip DAC in DACs list?

Overlay and driver are already in the kernel.
Thank you!

![1701_600](https://user-images.githubusercontent.com/43340836/133620313-f7feecdd-2702-4a35-89cf-f6ae874a924b.jpg)

**Reply by rern** [reply]
Done.

**Comment by clriis** [comment]
I started using Rune Audio 5 years ago on a Rpi 3B with Allo Boss I2S DAC. I found it to play music of high quality for my needs (320 kbit MP3, 25000 tracks and high-res radio) and also to be stable and responsive.

After ½ a year I switched to Volumio because I found the user interface to be more polished and friendly + it seemed that Rune Audio was kind of hibernating.

Over the years I've been glancing for alternatives because Volumio has become very rigid and with glitches every time there is an update. On the official Rune Audio website you really have to dig to find out that the project has evolved and I was very surprised when I discovered the 'rAudio 1' this week.

It is now up and running on my well used Rpi with a Rpi4 as NAS and I am really impressed:
- Superior music quality compared to Volumio
- Very fast and responsive (Boot duration 17 seconds!)
- A logical and coherent user interface
- A quick and stable Android web app
- Very easy to install and setup with comprehensive explanations for all settings.

Congratulations and Thank You!

**Comment by kw-sudo** [comment]
Thank you rern, you made my day!

Actually I‘ve switched from an Odroid C1+ (1. RuneAudio 0.3, 2. ArchLinuxARM with MPD, Lighttpd) to a RasPi 3B+ and with rAudio-1 it makes so much fun. I only need it as a network player (from NAS bitperfect/audiophile to my amp (USB, integrated DAC)).
